demo/src/utils.js

```diff
@@ -13,8 +17,12 @@ export async function createLoader(type, infoObj) {
       const loader = await createZarrLoader(infoObj);
       return loader;
     }
-    // These all resolve to the 'tiff' case.
-    case 'static tiff':
+    case 'ome-zarr': {
+      const reader = await OMEZarrReader.fromUrl(infoObj.url);
+      const { loader } = await reader.loadOMEZarr();
+      return loader;
+    }
+    case 'static tiff': // These all resolve to the 'tiff' case.
```

Comment:
When I got bioformats/OMETiffLoader working, I transitioned everything out of the old paradigm.  Could we do this for the Spraggins Kidney data?  Or can you file an issue for it if it's too much for right now?  The IMS data, I understand, is not really OME but I don't see why the Kidney data couldn't be.

Reply:
What is the old paradigm you are referring to? Just resolving to `tiff`?

Reply:
The pre-`OME-TIFF` pyramids I had.

Reply:
So, I think, yes.